### ISSUE TITLE
`yanu-cli` command not found on Android

### ISSUE THREAD

**Issue by StevenBeach33** [nozwock/yanu#46]
![Screenshot_20240118-023342.png](https://github.com/nozwock/yanu/assets/107557824/d9d69040-aa98-4a17-9550-04d86c2e273c)

Maybe script is broken

**Comment by nozwock** [comment]
Did you read the Installation instructions? `yanu-cli` is deprecated. Use `yanu` instead of `yanu-cli` now.

Maybe I should rephrase it to make it more obvious.

Example:
```
yanu pack \
              --controlnca './base+update.xxxxxx/patchdata/control.nca' \
              --titleid 'xxxxxxxxxxxxxxxx' \
              --romfsdir './base+update.xxxxxx/romfs' \
              --exefsdir './base+update.xxxxxx/exefs'
```
